### PR TITLE
Rename solrizer config

### PR DESCRIPTION
**Pull request by cbeer**
Includes #170

### CODE REVIEW

File: lib/dor/config.rb [hunk 1]
```diff
@@ -55,8 +55,10 @@ def make_rest_client(url, cert = Config.ssl.cert_file, key = Config.ssl.key_file
     end
 
     def make_solr_connection(add_opts = {})
-      opts = Config.solrizer.opts.merge(add_opts).merge(
-        :url => Config.solrizer.url
+      c = Dor::Config.solr || Dor::Config.solrizer
```
Comment (jmartin-sul) [inline lib/dor/config.rb]: might be nice to leave a comment somewhere about supporting the deprecated config namespace, and/or give users of it a deprecation warning.

File: spec/dor/assembleable_spec.rb [hunk 2]
```diff
@@ -12,8 +12,7 @@ class AssembleableItem < ActiveFedora::Base
 
     Dor::Config.push! do |config|
       config.suri.mint_ids false
-      config.gsearch.url  'http://solr.edu/gsearch'
-      config.solrizer.url 'http://solr.edu/solrizer'
+      config.solr.url 'http://solr.edu/solrizer'
```
Comment (jmartin-sul) [inline spec/dor/assembleable_spec.rb]: i propose `config.solr.url 'http://solr.edu/solr'`for consistency :)

File: spec/dor_config.rb [hunk 3]
```diff
@@ -22,11 +22,7 @@
     catalog.url 'http://example.edu/catalog/mods'
   end
 
-  gsearch do
-    rest_url 'https://example.edu/gsearch/rest'
-    url      'https://example.edu/solr/gsearch'
-  end
-  solrizer.url     'https://example.edu/solr/solrizer'
+  solr.url     'https://example.edu/solr/solrizer'
```
Comment (jmartin-sul) [inline spec/dor_config.rb]: same comment as the other example.  i mean not a big deal if you'd prefer not, but more consistent w/ real solr urls too.

Comment (cbeer) [inline spec/dor_config.rb]: I noticed there are existing tests that expect `https://example.edu/solr/solrizer`, and it wouldn't surprise me if there's other HTTP mocking too. I think it's a great idea, but I'd certainly prefer to defer it to a different pull request.

Comment (jmartin-sul) [inline spec/dor_config.rb]: ok, makes sense

File: lib/dor/config.rb (hunk 1 above)
Comment (cbeer) [inline lib/dor/config.rb]: :ok_hand: I've added a deprecation warning to the `after configure` callback.

**Pull request merged**